include/exadg/incompressible_navier_stokes/time_integration/time_int_interpolate_analytical_solution.h

```diff
@@ -123,9 +123,6 @@ class TimeIntInterpolateAnalyticalSolution : public TimeIntBDF<dim, Number>
   void
   do_timestep_solve() final
   {
-    velocity_np.zero_out_ghost_values();
-    pressure_np.zero_out_ghost_values();
-
```

Comment:
Why is this change made here in this PR? Right now, I do not see the connection to the other refactoring.